corehq/ex-submodules/casexml/apps/phone/tests/test_extension_indexes.py

```diff
@@ -97,6 +97,7 @@ def __new__(mcs, name, bases, dict):
             tests_to_run = run_single_tests
 
         for test in tests_to_run:
+            continue
```

Comment:
was this intentionally included? I think this would cause these tests to not run in py2 if merged as is

Reply:
This is known to fail in py3.